syntax = "proto3";
package grpcnative;
option java_package = "sb.rpc.grpc2";
option java_outer_classname = "OrderServiceProto2";
option java_multiple_files = true;

// The greeting service definition.
service OrderService2 {
    // Sends a greeting
    rpc getOrderDetail (OrderRequest2) returns (OrderDetail2) {}
}

message OrderRequest2 {
    string orderId = 1;
}

message OrderDetail2 {
    string name = 1;
    uint64 date = 2;
    double price = 3;
    string buyer = 4;
    string seller = 5;
}